src/main/java/com/powsybl/iidm/modification/topology/RemoveHvdcLine.java

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.modification.topology;
```

Comment:
Why  this package
Already `modifications` package exists

Reply:
Sorry
Ignore this remark

Reply:
hack copy/paste from core